spec/classes/katello_install_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe 'katello::install' do
+  let :pre_condition do
+      [
+        'include apache',
+      ]
+  end
+  let :facts do
+    {
+      :concat_basedir         => '/tmp',
+      :interfaces             => '',
+      :osfamily               => 'RedHat',
+      :operatingsystem        => 'CentOS',
+      :operatingsystemrelease => '6.5',
+      :fqdn                   => 'host.example.org',
+    }
+  end
+
+  context 'should install ostree package on enable_ostree == true' do
+    let(:pre_condition) do
+      [
+        'class {"katello":' \
+          'enable_ostree => true,' \
+        '}'
+      ]
+    end
+    it { expect contain_package("tfm-rubygem-katello-ostree").with_ensure('installed') }
+  end
+
+  context 'should NOT install ostree package on enable_ostree == false' do
+    let(:pre_condition) do
+      [
+        'class {"katello":' \
+          'enable_ostree => false,' \
+        '}'
+      ]
```

Comment:
I would do this like https://github.com/Katello/puppet-pulp/blob/master/spec/classes/pulp_config_spec.rb#L62